ols/app/main.py

```diff
@@ -8,7 +8,6 @@
 
 app = FastAPI()
 
-# config = load_config(os.environ.get("OLS_CONFIG_FILE","olsconfig.yaml"))
```

Comment:
this isn't a leftover, it's a future thing we'll need when we shift to config.yaml based configuration.  (Well we won't necessarily need exactly this, we'll probably pass the config filename as an argument rather than loading it as an env var, but that's something for @syedriko to work out/propose).

for now it's ok to remove this, but just calling it out for @syedriko's awareness.